src/cmdstan/io/json/json_data.hpp

```diff
@@ -119,6 +120,43 @@ class json_data : public stan::io::var_context {
     return empty_vec_r_;
   }
 
+  /**
+   * Read out a vector of complex numbers from the specifed name.
+   * @param name Name of Variable.
```

Comment:
This needs `@return` doc.

It should also have some light doc on the presumed input format or a pointer to where it's described.  Otherwise, the code's too cryptic.